### DISCUSSION TITLE
Daily Limit - suggestion

### DISCUSSION THREAD

**Discussion by orock45** [syt0r/Kanji-Dojo#77] [General]
Hi there!

I've set a daily limit, and currently, we have two options: study and review. I was wondering if you could provide advanced settings for:

Study writing limit
Study reading limit
Review writing limit
Review reading limit

The aim is to enhance improvement in each area separately. (I'm better at reading than at writing...)

Best,
Jacques

**Comment by syt0r** [comment]
I agree, this would be nice to have. I guess limit dialog will become separate screen to support this
